drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/host_rpu_umac_if.h

```diff
@@ -1391,6 +1392,23 @@ struct nrf_wifi_umac_cmd_mcast_filter {
 } __NRF_WIFI_PKD;
 
 
+/**
+ * struct nrf_wifi_umac_cmd_change_macaddr - Set Mac Address.
+ *
+ *    - This has to be used when interface is down.
+ *
+ * @umac_hdr: Refer to &struct img_umac_hdr.
+ * @mac_addr : Mac_addr to be programmed
+ *
+ */
+struct nrf_wifi_umac_change_macaddr_info {
```

Comment:
we can drop the `_info`

Reply:
As discussed, This we are keeping it as it is.